pomodoro.sh

```diff
@@ -80,48 +90,62 @@ chiming_with_input () {
     done
     clear_line
 }
+
 single_pomodoro_run () {
-    echo "Starting pomodoro $1"
+    echo "Pomodoro $1"
     work
     chiming_with_input
+    clear_line 2
     log "$1"
     rest
     chiming_with_input
 }
 
-show_help () {
-    echo "pomodoro: "
-    echo " This runs pomodoro from your terminal"
-    echo " During a count down, you can press p to pause/unpause the program"
-    echo " You can also press q to quit the program"
-    echo " -h: Show help file"
-    echo " -p <arg>: Set time for actual work"
-    echo " -r <arg>: Set time for rest"
-}
-
 rename_window_in_tmux () {
     if [[ -n $TMUX ]]; then
         tmux rename-window "pomodoro"
     fi
 }
 
+view_logs () {
+    # show the day's logs when called
+    # should add support to view other days logs
+    cat "$LOG_FILENAME"
+}
+
 log () {
     mkdir -p $LOG_DIR
-    FILENAME=$LOG_DIR$(date +"%F").log
-    touch "$FILENAME"
+    touch "$LOG_FILENAME"
     read -r -p 'Work done: ' work
-    echo 'Pomodoro' "$1" ':' "$work" >> "$FILENAME"
+    clear_line
+    echo -e '\tWork done: ' $work
+    echo 'Pomodoro' "$1" ':' "$work" >> "$LOG_FILENAME"
+}
+
+show_help () {
+    echo "pomodoro: "
+    echo " This runs pomodoro from your terminal"
+    echo " During a count down, you can press p to pause/unpause the program"
+    echo " You can also press q to quit the program"
+    echo " -h: Show help file"
+    echo " -p <arg>: Set time for actual work"
+    echo " -r <arg>: Set time for rest"
+    echo " -l: Daily retrospection (Show work done during the day)"
```

Comment:
I'd suggest using a `heredoc` here since it's easier to write and makes more sense in terms of formatting. Here's something that works(tested on my machine running FreeBSD using bash v5.0.11):

```
show_help() {
    cat <<EOF
Copyright (C) 2009: John Nduli K.

Pomodoro:

This runs pomodoro from your terminal
During a count down, you can press p to pause/ unpause the program
You can also press q to quit the program

-h: Show help file
-p <arg>: Set time for actual work
-r <arg>: Set time for rest
-l: Daily retrospection (Show work done)
EOF
}
```

Note that I've added a copyright at the top.